drivers/pwm/pwm_renesas_ra8.c

```diff
@@ -0,0 +1,590 @@
+	}
+}
+
+#endif /* CONFIG_PWM_CAPTURE */
+
+static const struct pwm_driver_api pwm_ra8_driver_api = {
+	.get_cycles_per_sec = pwm_ra8_get_cycles_per_sec,
+	.set_cycles = pwm_ra8_set_cycles,
+#ifdef CONFIG_PWM_CAPTURE
+	.configure_capture = pwm_ra8_configure_capture,
+	.enable_capture = pwm_ra8_enable_capture,
+	.disable_capture = pwm_ra8_disable_capture,
+#endif /* CONFIG_PWM_CAPTURE */
+};
+
+static int pwm_ra8_init(const struct device *dev)
+{
+	struct pwm_ra8_data *data = dev->data;
+	const struct pwm_ra8_config *cfg = dev->config;
+	int err;
+
+	if (!device_is_ready(cfg->clock_dev)) {
+		LOG_ERR("clock control device not ready");
+		return -ENODEV;
+	}
+
+	err = clock_control_on(cfg->clock_dev, (clock_control_subsys_t)&cfg->clock_subsys);
+	if (err < 0) {
+		LOG_ERR("Could not initialize clock (%d)", err);
+		return err;
+	}
+
+	err = pinctrl_apply_state(cfg->pincfg, PINCTRL_STATE_DEFAULT);
+	if (err) {
+		LOG_ERR("Failed to configure pins for PWM (%d)", err);
+		return err;
+	}
+
+#if defined(CONFIG_PWM_CAPTURE)
+	data->fsp_cfg.p_callback = fsp_callback;
+	data->fsp_cfg.p_context = dev;
+#endif /* defined(CONFIG_PWM_CAPTURE) */
+
+	data->fsp_cfg.p_extend = &data->extend_cfg;
+
+	err = R_GPT_Open(&data->fsp_ctrl, &data->fsp_cfg);
+	ZEPHYR_ERROR_LOG(err);
+	return 0;
+}
+
+#define _ELC_EVENT_GPT_CAPTURE_COMPARE_A(channel) ELC_EVENT_GPT##channel##_CAPTURE_COMPARE_A
+#define _ELC_EVENT_GPT_COUNTER_OVERFLOW(channel)  ELC_EVENT_GPT##channel##_COUNTER_OVERFLOW
+
+#define ELC_EVENT_GPT_CAPTURE_COMPARE_A(channel) _ELC_EVENT_GPT_CAPTURE_COMPARE_A(channel)
+#define ELC_EVENT_GPT_COUNTER_OVERFLOW(channel)  _ELC_EVENT_GPT_COUNTER_OVERFLOW(channel)
+
+#ifdef CONFIG_PWM_CAPTURE
+#define PWM_RA_IRQ_CONFIG_INIT(index)                                                              \
+	do {                                                                                       \
+                                                                                                   \
+		IRQ_CONNECT(DT_INST_IRQ_BY_NAME(index, gtioca, irq),                               \
+			    DT_INST_IRQ_BY_NAME(index, gtioca, priority),                          \
+			    gpt_capture_compare_a_isr, NULL, 0);                                   \
+		IRQ_CONNECT(DT_INST_IRQ_BY_NAME(index, overflow, irq),                             \
+			    DT_INST_IRQ_BY_NAME(index, overflow, priority),                        \
+			    gpt_counter_overflow_isr, NULL, 0);                                    \
+	} while (0)
+
+#else
+#define PWM_RA_IRQ_CONFIG_INIT(index)
+#endif /* CONFIG_PWM_CAPTURE */
+
+#define PWM_RA8_INIT(index)                                                                        \
+	PINCTRL_DT_INST_DEFINE(index);                                                             \
+	static const gpt_extended_cfg_t g_timer1_extend_##index = {                                \
+		.gtioca =                                                                          \
```

Comment:
the brace should be inline here, same as what you did with `g_timer1_extend_##index`, `clock_subsys` below and others

Reply:
Thanks for your comment, this change because we ran ClangFormat for this source

Reply:
got it, yeah was a bit of a nit, clang-format is a bit inconsistent on these